src/main/java/de/ait/secondlife/security/config/SecurityConfig.java

```diff
@@ -33,17 +34,23 @@ public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Excepti
                 .sessionManagement(x -> x
                         .sessionCreationPolicy(SessionCreationPolicy.STATELESS))
                 .authorizeHttpRequests(x -> x
-//                        .requestMatchers(HttpMethod.GET, "/v1/auth/admin/logout").hasRole("ADMIN")
-//                        .requestMatchers(HttpMethod.GET, "/v1/auth/user/logout").hasRole("USER")
-//                        .requestMatchers(HttpMethod.POST,
-//                                "/v1/auth/admin/login",
-//                                "/v1/auth/admin/access",
-//                                "/v1/auth/user/login",
-//                                "/v1/auth/user/access"
-//                        ).permitAll()
-//                        .requestMatchers(HttpMethod.GET, "/swagger-ui/**", "/v3/api-docs/**").permitAll()
-//                        .requestMatchers(HttpMethod.POST, "/v1/users/register").permitAll()
-                        .anyRequest().permitAll())
+                        .requestMatchers(HttpMethod.GET, "/v1/auth/admin/logout").hasRole("ADMIN")
+                        .requestMatchers(HttpMethod.GET, "/v1/auth/user/logout").hasRole("USER")
+                        .requestMatchers(HttpMethod.POST,
+                                "/v1/auth/admin/login",
+                                "/v1/auth/admin/access",
+                                "/v1/auth/user/login",
+                                "/v1/auth/user/access"
+                        ).permitAll()
+                        .requestMatchers(HttpMethod.GET, "/swagger-ui/**", "/v3/api-docs/**").permitAll()
+                        .requestMatchers(HttpMethod.POST, "/v1/users/register").permitAll()
+                        .requestMatchers(HttpMethod.GET, "/v1/categories").hasRole("ADMIN")
+                        .requestMatchers(HttpMethod.GET, "/v1/categories/**").hasRole("ADMIN")
```

Comment:
if "../{category-id}" works, I think it is better to use the same pattern for other endpoints

Reply:
Done